Dockerfile.py3.aws

```diff
@@ -0,0 +1,165 @@
+# Build the docker image used to run the scripts
+# to continuously update our docker files.
+#
+# The context for this docker file should be the root of the kubeflow/testing repository.
+FROM ubuntu:18.04
+
+RUN apt-get update -y && \
+    apt-get install -y curl git python3.8 python3-pip wget && \
+    ln -sf /usr/bin/python3.8 /usr/bin/python
+
+RUN python3.8 -m pip install \
+    filelock \
+    fire \
+    google-api-python-client \
```

Comment:
Is there's a Dockerfile.py3?

If so, what's the difference between Dockerfile.py3 vs Dockerfile.py3.aws? 

If there's minior changes, I would suggest to change original dockerfile. Otherwise, you need to remove these GCP stuff.

Reply:
Yes, there's [Dockerfile.py3](https://github.com/kubeflow/testing/blob/master/images/Dockerfile.py3).

There's some changes in AWS Dockerfile, so I clean up GCP resources and make our own Dockerfile.